### PR TITLE
Update docker build

### PR DESCRIPTION
**Pull request by plarocque4**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (KeeyanGhoreshi) [inline gradlew:47]: Is 64mb enough?  I had thought CRD was bigger than that.

Comment (plarocque4) [reply gradlew:47]: That was something that was automatically added when I ran the "gradle wrapper --gradle-version 5.6.4" command.
I'll remove the default option and put it back to an empty string.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Thu Jul 19 13:08:23 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```
Comment (yuliumitre) [inline gradle/wrapper/gradle-wrapper.properties:3]: In CRD prerequisite, we are saying Gradle 6.3, wondering why we are using 5.6.4 in Docker?

Comment (plarocque4) [reply gradle/wrapper/gradle-wrapper.properties]: I updated to 6.3

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**